lib/keyrod/cli.rb

```diff
@@ -0,0 +1,44 @@
+require 'thor'
+
+module Keyrod
+  class CLI < Thor
+    desc 'token', 'Get login token for FedCloud site'
+    option :'site',
+           required: true,
+           default: Keyrod::Settings['fedcloud']['site'],
+           type: :string,
+           desc: 'EGI FedCloud Site',
+           aliases: '-s'
+    option :'access-token',
+           required: false,
+           default: Keyrod::Settings['fedcloud']['access_token'],
+           type: :string,
+           desc: 'Access token for authentication',
+           aliases: '-a'
+    option :'group',
+           required: false,
+           default: Keyrod::Settings['fedcloud']['group'],
+           type: :string,
+           desc: 'Group to join',
+           aliases: '-g'
+    option :'fall-back-to-interactive',
```

Comment:
Either use `fall-back` or `fallback` but don't mix it.
Maybe I would even prefere the name `interactive-fallback` to simplify it a bit. And use it in the configuration file as well so it looks the same.